style.css

```diff
@@ -0,0 +1,87 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-style: normal;
+  font-family: Poppins, sans-serif;
+}
+
+.container {
+  max-width: 390px;
```

Comment:
I would recommend working from the `min-width: 375px` upwards. Currently, when the size of the page exceeds the `390 px`, your layout starts to break with white margins on the right side of your applications. Remember, nowadays mobile phones have got bigger, so having a flexible screen size for your application is a must.
Have a look at your website when the size increases beyond `390px` but is still within the mobile version. ⬇️
![image](https://user-images.githubusercontent.com/33268927/205086950-da2b5ccf-4aac-4bb7-a1f2-35c4028d0f34.png)